context/src/main/java/io/grpc/Deadline.java

```diff
@@ -55,6 +55,9 @@ public static Ticker getSystemTicker() {
    * Create a deadline that will expire at the specified offset based on the {@link #getSystemTicker
    * system ticker}.
    *
+   * <p>If the given offset is extraordinarily long, say 100 years, the actual deadline created
```

Comment:
s/offset/duration/

Reply:
Actually the first sentence mentioned offset 
`* Create a deadline that will expire at the specified offset`

Reply:
Okay. This is fine then.